### PR TITLE
added 4 new questions,  there were a repeat of two questions for ques…

### PR DESCRIPTION
**Pull request by swordwielder**
…tion Q5 and question Q11, removed both of them and fixed the numbering

### CODE REVIEW

File: nosql/nosql-quiz.md [hunk 1]
```diff
@@ -138,116 +138,134 @@
  - [ ] MongoDB
  - [ ] Bigtable
 
-#### Q24. You need to implement the simples possible scalable, in-memory cache for your AWS application. Which service do you select?
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:141]: sorry but what is the reason of removing this test? I can't find any related duplication FYI

File: nosql/nosql-quiz.md [hunk 2]
```diff
@@ -138,116 +138,134 @@
...
  - [ ] medium
  - [x] a single bit
+
+#### Q42. You need to store an unordered collection of name-value pairs with differing data types in DynamoDB.  Which data type should you choose?
+ - [x] map
+ - [ ] set
+ - [ ] list
+ - [ ] stack
+
+ #### Q43. Which statement retrieves an item from the MusicCollection table in DynamoDB?
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:255]:
```suggestion
#### Q43. Which statement retrieves an item from the MusicCollection table in DynamoDB?
```

File: nosql/nosql-quiz.md [hunk 3]
```diff
@@ -138,116 +138,134 @@
...
+ - [ ] aws dynamodb query --table-name MusicCollection --key file://key.json
+ - [x] aws dynamodb get-item --table-name MusicCollection --key file://key.json
+ - [ ] aws dynamodb select --table-name MusicCollection --key file://key.json
+ - [ ] aws dynamodb put-item --table-name MusicCollection --key file://key.json
+
+ #### Q44. Database availability is measured by which metric?
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:261]:
```suggestion
#### Q44. Database availability is measured by which metric?
```

File: nosql/nosql-quiz.md [hunk 4]
```diff
@@ -138,116 +138,134 @@
...
+ - [ ] the amount of service calls
+ - [ ] the number of minutes
+ - [ ] the amount of service costs
+ - [ ] the number of nines
+ 
+  #### Q44. You need to encrypt data at rest in DynamoDB.  Which action do you take?
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:267]:
```suggestion
 #### Q44. You need to encrypt data at rest in DynamoDB.  Which action do you take?
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Left some comments @swordwielder BTW great job!

**Review by swordwielder — COMMENTED** [review]
please check with Question, there was a spelling error with that one too but the choice are the same.

File: nosql/nosql-quiz.md (hunk 1 above)
Comment (Ebazhanov) [reply nosql/nosql-quiz.md:141]: ahh, ok I see it now, good finding! FYI there is only different answers :grimacing:

**Review by Ebazhanov — APPROVED** [review]
Thanks, @swordwielder great improvements!

**Pull request merged**